library/pathname/empty_spec.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path('../../spec_helper', __dir__)
+require 'pathname'
+
+describe "Pathname#empty?" do
+  ruby_version_is "2.4" do
```

Comment:
We usually swap those two lines, so that it's clear the whole block until `^end` only applies for Ruby 2.4+, and that the method does not exist before Ruby 2.4.